### ISSUE TITLE
Should update the login example using the latest form validation design spec

### ISSUE THREAD

**Issue by huayunh** [pxblue/react-native-workflows#21]
#### Describe the desired behavior
The login example has some minor interactions and designs that do not fit with the latest form validation guidance 

https://pxblue.github.io/patterns/forms#design-specifications

#### Describe the current behavior

It does not

#### Is this request related to a current issue?

Related to comments from PR https://github.com/pxblue/react-native-workflows/pull/15

**Comment by emclaug2** [comment]
Pay attention to the spacing specifications.

**Comment by joebochill** [comment]
I have spent the better part of two days trying to clean up the styles around this workflow and it is a giant can of worms. I got it to a point where it's starting to work really nice for iOS, but Android is terrible. The whole codebase is really messy and in need of a serious refactoring (creating reusable styles, wrapper components, etc.) - probably at least a week's worth of time. More work than I care to spend right now given our other priorities...I pushed up my work in progress to the [experiment/style-fixes](https://github.com/pxblue/react-native-workflows/tree/experiment/style-fixes) branch.
